gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/TraversalInterruptionComputerTest.java

```diff
@@ -86,7 +97,7 @@ public void shouldRespectThreadInterruptionInVertexStep() throws Exception {
 
         // total time for test should not exceed 5 seconds - this prevents the test from just hanging and allows
         // it to finish with failure
-        assertThat(startedIterating.await(5000, TimeUnit.MILLISECONDS), CoreMatchers.is(true));
+        assertThat(startedIterating.await(1000, TimeUnit.MILLISECONDS), CoreMatchers.is(true));
```

Comment:
When I determined the timing issue was related to the traversal finishing first as opposed to it not having enough time to start I figured I could experiment with lowering this but it was admittedly a shot in the dark at what the "right" number is.